videoanalyst/data/transformer/transformer_impl/random_crop_transformer.py

```diff
@@ -18,13 +19,27 @@ class RandomCropTransformer(TransformerBase):
     Hyper-parameters
     ----------------
 
+    context_amount: float
+        the context factor for template image
+    max_scale: float
+        the max scale change ratio for search image
+    max_shift:  float
+        the max shift change ratio for search image
+    max_scale_temp: float
+        the max scale change ratio for template image
+    max_shift_temp:  float
+        the max shift change ratio for template image
+    z_size: int
+        output size of template image
+    x_size: int
+        output size of search image
     """
     default_hyper_params = dict(
-        context_amount=0.5,
+        context_amount=1.0,

```

Comment:
Again, the default value should be conformed with the definition in crop_track_pair.

Reply:
fixed